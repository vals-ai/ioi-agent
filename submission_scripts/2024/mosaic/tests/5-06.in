version https://git-lfs.github.com/spec/v1
oid sha256:c00e9d18195f13386fcc28cfa834914d3b6f004b8f342a67e386a428067af847
size 5955348
