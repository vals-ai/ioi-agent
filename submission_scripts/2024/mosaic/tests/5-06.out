version https://git-lfs.github.com/spec/v1
oid sha256:bbeb925aa0fdc0ec76fea0d81cd7c11c736e60f2a56653cd6070156607bfc7dc
size 1157552
